backend/src/main/java/com/benine/backend/camera/CameraController.java

```diff
@@ -44,6 +42,22 @@ public void addCamera(Camera camera) {
     highestIdInUse++;
     cameras.add(camera);
   }
+  
```

Comment:
Yes you are right they could be replaced with the call to ServerController.getInstance(), but since they are used in this class a few times. I though it would be a cleaner to have a possibility to just call getDatabase() for example. Since they are only used in this class I can make them private methods.